package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -87,125 +100,143 @@ export interface SpotlightTourProviderProps {
   steps: TourStep[];
 }
 
+const DEFAULT_FLOATING_CONFIGURATIONS: FloatingProps = {
```

Comment:
Nit: Maybe change this to `FLOATING_PROPS` or `DEFAULT_FLOATING_PROPS`? I'd try to keep it simple, but your call 😬

Reply:
Sure i like ```DEFAULT_FLOATING_PROPS ```